tools/scripts/CaptureDevHomeLogs.ps1

```diff
@@ -0,0 +1,18 @@
+# Quick script to capture log files and create a zip file for upload to GitHub
+# Currently this only creates one based on Date. We could add a time, so that 
+# the ISV can have more than one Zip file
+
+$MyZip=$(Get-Date -Format yyyy-MM-dd) + "DevHome.Zip"

```

Comment:
```suggestion
$MyZip=$(Get-Date -Format yyyy-MM-dd-HHmmss_) + "DevHome.zip"
```
This adds hour in 24 hour format, minutes, and seconds to the filename, which while not guaranteed to be unique, should be sufficient to avoid collisions with subsequent runs of the script, and is a sorted, human-readable identifier.

Result is something like: 2024-06-04-131303_DevHome.zip